storage/zfs/pkg.yaml

```diff
@@ -27,6 +27,13 @@ steps:
         cp -r /rootfs/ /extensions-validator-rootfs/rootfs
         cp /pkg/manifest.yaml /extensions-validator-rootfs/manifest.yaml
         /extensions-validator validate --rootfs=/extensions-validator-rootfs --pkg-name="${PKG_NAME}"
+    sbom:
+      outputPath: /rootfs/usr/local/share/spdx/zfs.spdx.json
```

Comment:
Again, here we only copy modules built in pkgs, so it's handled like NVIDIA nonfree: build an SBOM in pkgs, copy to the appropriate location here. Maybe it'd be more complex if a wrapper is added here (like nvidia-fabric-manager here, which has a Go-based wrapper process)

Reply:
Removed.